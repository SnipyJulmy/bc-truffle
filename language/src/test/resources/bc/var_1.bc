a = 1
b = 2
a
b
a = 3
a
b
b = 5
a
b
c = 10
a
b
c
d
a
b
d
d = 467
d
a
b
d
halt
