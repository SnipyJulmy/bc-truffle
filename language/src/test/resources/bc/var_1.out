1
2
3
2
3
5
3
5
10
0
3
5
0
467
3
5
467
